e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.ts

```diff
@@ -219,6 +221,21 @@ describe('Pricing modal', () => {
         cy.get('#UpgradeButton').should('exist');
     });
 
+    it('should show Upgrade button in global header for admin users on monthly professional', () => {
+        const subscription = {
+            id: 'sub_test1',
+            product_id: 'prod_2', // monthly professional
+            is_free_trial: 'false',
+        };
+        simulateSubscription(subscription);
+        cy.apiLogout();
+        cy.apiAdminLogin();
+        cy.visit(urlL);
+
+        // * Check that Upgrade button shows for admins
+        cy.get('#UpgradeButton').should('exist');
+    });
+
```

Comment:
The Upgrade button in the channel header should not show for monthly users as they are already in the paid plan

Reply:
Don't we want to encourage monthly users to convert to yearly? "upgrade" may not be the best word to describe it but it might be worth leaving?

Reply:
Given it's the same plan they are not technically upgrading - this is the button that we show for Free users. The message to switch billing method is in the System Console and a dismissible global banner. cc/ @johndavidlugtu to make sure I didn't miss anything

Reply:
+1 on @marianunez 's comment. There's going to be a persistent banner in the System Console prompting them to _switch_ their billing frequency